### PR TITLE
Fix/lint issues

### PR DESCRIPTION
**Pull request by rafaelsousa**
### Description:

Fixes #(issue)

### Checklist:

- [x] Tests included.
- [x] Lint and formatter run with no errors.
- [x] Tests all pass.

#### Where applicable:

- [ ] Public APIs match existing style.
- [ ] Any breaking changes have a deprecation path or have been discussed.
- [ ] Updated the vendor folder (using `go mod vendor`).

### CODE REVIEW

**Review by andydotxyz — CHANGES_REQUESTED** [review]
We can't use os.ReadFile yet as we support back to Go 1.14 and it was not present then.

**Comment by Jacalz** [comment]
It looks like there still are a few places where ioutil has been removed. That will need to be added back.

**Comment by rafaelsousa** [comment]
> We can't use os.ReadFile yet as we support back to Go 1.14 and it was not present then.

The os.Readfile references were reverted.

File: modules/systray/main.go [hunk 1]
```diff
@@ -206,10 +206,14 @@ func (t *tray) RegisterStatusNotifierItem(service string, sender dbus.Sender) (e
 func (t *tray) RegisterStatusNotifierHost(service string, sender dbus.Sender) (err *dbus.Error) {
 	log.Println("Register Host", service, sender)
 
-	watcher.Emit(t.conn, &watcher.StatusNotifierWatcher_StatusNotifierHostRegisteredSignal{
+	e := watcher.Emit(t.conn, &watcher.StatusNotifierWatcher_StatusNotifierHostRegisteredSignal{
 		Path: dbus.ObjectPath(service),
 		Body: &watcher.StatusNotifierWatcher_StatusNotifierHostRegisteredSignalBody{},
 	})
+	// TODO: See the need of returning this error
```
Comment (andydotxyz) [inline modules/systray/main.go:213]: Is this TODO supposed to be there? Not sure I understand the comment

**Review by andydotxyz — COMMENTED** [review]
Looking great thanks - just checking about the TODO

**Review by andydotxyz — APPROVED** [review]

**Pull request merged**